### ISSUE TITLE
HomeBridge dimmables wrong values

### ISSUE THREAD

**Issue by youridijk** [youridijk/homebridge-kaku-ics2000#28]
In #12 @RonaldR and @martienvd pointed this out. In my case the following happens:
- Dim a dimmable to 100% only goes to 39% as 100/255  = 39% 
- On max brightness, HomeKit shows a brightness of 255% instead of 100%

I'll wait and see if HomeBridge team fixes this. Please open an issue at their repo.

**Comment by RonaldR** [comment]
@youridijk is `maxBrightness: 15` in https://github.com/youridijk/ICS-2000-NodeJS/blob/main/src/kaku/DeviceConfigs.ts not the issue?

**Comment by rickklh** [comment]
Same issue here, I think that the issue indeed started when updated to iOS 17. I ordered ah older iPad today i hope it has iOS 16.x installed. I can try it out if its in iOS.

In Home it shows the correct value in the small overview after comparing it to the default KAKU app but when you open the dimmables to set the value it is shows very low values. 

example: overview 40% open 6%
overview 47% open 7%
overview 13% open 2%

you can change it a bit but it falls back down but it will change. When I try to up the 13% / 2% it falls back stays on 11% and the overview shows 73%.

**Comment by youridijk** [comment]
Yes, same problems here as you describe. Definitely an issues with homebridge, but can’t seem to find an issue about it on their GitHub or somewhere else. Feels weird, can’t be that it’s just us. Anyone experiencing same issues with other plugins?

**Comment by youridijk** [comment]
I fixed the issue by manually convert the 0-100 brightness range from Homebridge to a 0-255 range KAKU uses. Don't really like the solution but works fine. Should be backwards compatible with previous iOS versions. I just released a new version of this plugin with the fix. If after the update the issue isn't gone, please empty your cache.

**Comment by rickklh** [comment]
Works, great! Only thing now is that the value is wrong other way around. When open it is correct for example 30% closed 200% so that looks a bit weird but it works again and no flyback down.

**Comment by martienvd** [comment]
thanks, works perfectly!

**Comment by youridijk** [comment]
@rickklh  Hmm I'm not experiencing it after clearing the cache. Maybe it will help you too.

**Comment by rickklh** [comment]
> @rickklh Hmm I'm not experiencing it after clearing the cache. Maybe it will help you too.

I did that after the first post because after the update to 1.3.2 I didn’t work and cleared the cache of homebridge like you said. No issues again on iOS 16.4 only 17. 

Tried rebooting the whole system, no succes but simply after rebooting my phone the issue was gone!

so for anyone else having this problem, reboot your stuff might be the solution.

Thanks for your response and great homebridge plug-in. Much appreciated.
